compiler/circle-mpqsolver/src/core/Evaluator.test.cpp

```diff
@@ -18,6 +18,24 @@
 
 #include "Evaluator.h"
 
+#include "DataProvider.h"
+#include "TestHelper.h"
+
+TEST(CircleMPQSolverEvaluatorTest, verifyResultsTest)
+{
+  // create nn module
+  auto m = luci::make_module();
+  mpqsolver::test::models::AddGraph g;
+  g.init();
+  g.transfer_to(m.get());
+
+  mpqsolver::core::MAEMetric metric;
+  auto data = mpqsolver::test::data_utils::getSingleDataProvider();
```

Comment:
Not for this PR, but it would be better to rename this to `getAllZeroSingleDataProvider` for clarity.

```suggestion
  auto data = mpqsolver::test::data_utils::getSingleDataProvider();
```

Reply:
Ahh. Ok.